docs/components/ColorSwatchGroup/ColorSwatchGroup.tsx

```diff
@@ -13,82 +13,157 @@ interface ColorSwatchGroupProps {
   textBackgroundColor?: string;
   borderColor?: string;
   textColor?: string;
+  /** Hex code value of the theme (light or dark mode) this is used to help determine the text color of each swatch when opacity is involved
+   * Default is light theme #ffffff
+   */
+  theme?: string;
+}
+
+function toCamelCase(str: string) {
+  // Check if the string contains a dash followed by a number, if so, skip modification
+  if (/\-\d+%$/.test(str)) {
+    return str;
+  }
+  return str.replace(/-([a-z])/gi, function (g) {
+    return g[1].toUpperCase();
+  });
 }
 
 export const ColorSwatchGroup: React.FC<ColorSwatchGroupProps> = ({
   swatchData,
   borderColor,
   textBackgroundColor = 'transparent',
-  textColor = '#212529',
+  theme = '#ffffff',
```

Comment:
Looks like `textColor` has been removed here but its still in the `ColorSwatchGroupProps` enum